packages/strapi-plugin-content-manager/admin/src/containers/EditViewDataManagerProvider/utils/tests/testData.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
Why a JSON file?

Reply:
In other tests, we export a JS object. (rawData.js or data.js)

Reply:
I don't know I can change it to a js one.